pyodide_build/cli/xbuildenv.py

```diff
@@ -205,3 +205,31 @@ def _search(
         print(MetadataView.to_json(views))
     else:
         print(MetadataView.to_table(views))
+
+
+@app.command("install-emscripten")
+def _install_emscripten(
+    version: str = typer.Option(
+        None,
+        help="Emscripten SDK Version (default: PYODIDE_EMSCRIPTEN_VERSION from xbuildenv)",
```

Comment:
Variable name like `PYODIDE_EMSCRIPTEN_VERSION` is not user-friendly (actually, it is an implementation detail that user doens't know).

So maybe let's just replace the help help text to such as:

```
Emscripten version corresponding to the target Pyodide version
```